metadata/base/metadata-deployment.yaml

```diff
@@ -16,20 +16,21 @@ spec:
     spec:
       containers:
       - name: container
-        image: gcr.io/kubeflow-images-public/metadata:v0.1.8
+        image: gcr.io/tfx-oss-public/ml_metadata_store_server:0.14.0
         env:
           - name: MYSQL_ROOT_PASSWORD
             valueFrom:
               secretKeyRef:
                 name: metadata-db-secrets
                 key: MYSQL_ROOT_PASSWORD
-        command: ["./server/server",
-                  "--http_port=8080",
-                  "--mysql_service_host=metadata-db.kubeflow",
-                  "--mysql_service_port=3306",
-                  "--mysql_service_user=root",
-                  "--mysql_service_password=$(MYSQL_ROOT_PASSWORD)",
-                  "--mlmd_db_name=metadb"]
+        command: ["/bin/metadata_store_server"]
```

Comment:
I think current UI and SDK still depends on current HTTP server. So can you keep the existing server and add the gPRC server with a different port like 8081? We can remove existing server later.

Reply:
Besides the port, the existing server is still installed as part of the setup.

Reply:
Where is the container image `gcr.io/kubeflow-images-public/metadata:v0.1.8`deployed?

Can you make the port the same? All existing examples (and possible user code) depends on `kubeflow-metadata:8080` pointing to the `metadata:v0.1.8` server. To test it, I created https://github.com/kubeflow/metadata/pull/129, which is failing right now indicating a regression.